src/libs/actions/IOU.js

```diff
@@ -692,7 +692,7 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
     const optimisticIOUReport = ReportUtils.buildOptimisticIOUReport(recipientEmail, managerEmail, amount, chatReport.reportID, currency, preferredLocale, true);
 
     // This will be deprecated soon, in case the migration happens before this PR is merged we'll need to adjust the code here
-    const newSequenceNumber = Report.getMaxSequenceNumber(chatReport.reportID) + 1;
+    const newSequenceNumber = isNewChat ? 0 : Report.getMaxSequenceNumber(chatReport.reportID) + 1;
```

Comment:
I think the CREATED action will always have the sequence number as 0, so it should be 1 here? cc @mountiny.